news/7066.breaking.rst

```diff
@@ -0,0 +1,5 @@
+The ``del_import`` method has been removed from the ``PostGraphAPI``
```

Comment:
Shall we make the next release v6.0 then?

Reply:
Not quite yet please.

Reply:
Actually, I suspect the breaking change here is more of a formality than something really impactful. The method was not used in `PyInstaller` hooks nor the ones from `pyinstaller-hooks-contrib`. There is of course possibility of 3rd party users, but as this requires the use of `hook()` function, I doubt it is very common...

FWIW, we could leave `del_import` in and have it try to do its post-graph operation, just in case...